package.json

```diff
@@ -43,23 +43,23 @@
 	},
 	"devDependencies": {
 		"@rollup/plugin-node-resolve": "^13.0.6",
-		"@types/node": "^16.11.7",
+		"@types/node": "^16.11.9",
```

Comment:
- we don't want any changes to the `package.json` or `package-lock.json` in this branch